### PR TITLE
chore: fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./static,./CHANGELOG.md,./yarn.lock -L everytime,falsy,formattings,pres,substract`

### CODE REVIEW

**Comment by nathanlesage** [comment]
Hej, sorry for getting back to you so late. I like the initiative, but since I've in the meantime merged the CodeMirror 6 upgrade, there are a few merge conflicts. As soon as these are fixed I'll have a look over your PR and give more detailed feedback! Thanks!

**Comment by luzpaz** [comment]
Rebased

File: source/main.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ const filesBeforeOpen: string[] = []
 let zettlr: Zettlr|null = null
 
 /**
- * This variable is being used to determine if all servive providers have
+ * This variable is being used to determine if all survive providers have
```
Comment (nathanlesage) [inline source/main.ts:98]:
```suggestion
 * This variable is being used to determine if all service providers have
```

**Review by nathanlesage — APPROVED** [review]
Thank you for rebasing! And thanks for finding so many typos (although 26 files in such a large project is definitely not too bad).

There was one single wrong correction, but other than that, all is good!

**Pull request merged**